### PR TITLE
♻️ modern RepoCasesTable

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

**Comment by alex-wilmer** [comment]
needs to be updated to handle sets

File: src/packages/@ncigdc/modern_components/RepoCasesTable/utils.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+// @flow
```
Comment (Jephuff) [inline src/packages/@ncigdc/modern_components/RepoCasesTable/utils.js]: Is there any difference between this and the one in `tableModels/utils.js`?

File: src/packages/@ncigdc/modern_components/RepoCasesTable/RepoCasesTable.relay.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+import React from 'react';
+import { graphql } from 'react-relay';
+import { compose, withPropsOnChange } from 'recompose';
+import { withRouter } from 'react-router-dom';
+import { parse } from 'query-string';
+import {
+  parseIntParam,
+  parseFilterParam,
+  parseJSURLParam,
```
Comment (Jephuff) [inline src/packages/@ncigdc/modern_components/RepoCasesTable/RepoCasesTable.relay.js]: `parseJSURLParam`  has been renamed `parseJSONParam`

File: src/packages/@ncigdc/modern_components/RepoCasesTable/RepoCasesTable.js [hunk 3]
```diff
@@ -15,13 +14,20 @@ import RemoveFromRepositoryCaseSetButton from '@ncigdc/modern_components/setButt
...
   withSelectIds,
 )(
   ({
-    relay,
-    hits,
+    variables,
+    viewer: { repository: { cases: { hits } } },
     entityType = 'cases',
     tableColumns,
     filters,
```
Comment (Jephuff) [inline src/packages/@ncigdc/modern_components/RepoCasesTable/RepoCasesTable.js]: Seems like filters isn't passed in here? they are need for creating a new set, right now it always creates a set with  out filters.

File: src/packages/@ncigdc/modern_components/RepoCasesTable/utils.js (hunk 1 above)
Comment (alex-wilmer) [reply src/packages/@ncigdc/modern_components/RepoCasesTable/utils.js]: ok removing

File: src/packages/@ncigdc/modern_components/RepoCasesTable/RepoCasesTable.relay.js (hunk 2 above)
Comment (alex-wilmer) [reply src/packages/@ncigdc/modern_components/RepoCasesTable/RepoCasesTable.relay.js]: thanks 👍

File: src/packages/@ncigdc/modern_components/RepoCasesTable/RepoCasesTable.js (hunk 3 above)
Comment (alex-wilmer) [reply src/packages/@ncigdc/modern_components/RepoCasesTable/RepoCasesTable.js]: defaultFilters is coming from above, so renaming this

**Comment by alex-wilmer** [comment]
@Jephuff I think this is good now

**Pull request merged**